### PR TITLE
[16.0][MIG] mgmtsystem_audit: Migration to 16.0

### PR DESCRIPTION
**Pull request by FernandoRomera**

### CODE REVIEW

**Review by AntoniRomera — APPROVED** [review]

**Review by MRomeera — APPROVED** [review]
Perfect!

**Comment by FernandoRomera** [comment]
@max3903 
Once the previous PR (mgmtsystem_nonconformity) has been merged, we can proceed with this one.

File: mgmtsystem_audit/data/audit_automated_actions.xml [hunk 1]
```diff
@@ -33,7 +34,7 @@ Odoo
         <field name="model_id" ref="model_mgmtsystem_audit" />
         <field name="template_id" ref="template_email_prepare_to_audit" />
         <field name="trigger">on_time</field>
-        <field name="state">email</field>
+        <!-- <field name="state">email</field> -->
```
Comment (ivantodorovich) [inline mgmtsystem_audit/data/audit_automated_actions.xml]:
```xml
<field name="state">mail_post</field>
```

I think you want `mail_post` here, no?
Commenting it out will fallback to the default `object_write`, and that's not what we want here

See: https://github.com/odoo/odoo/commit/b3fc41cc44b251be37b6aaf2931179afeb126d69

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: mgmtsystem_audit/data/audit_automated_actions.xml (hunk 1 above)
Comment (ivantodorovich) [reply mgmtsystem_audit/data/audit_automated_actions.xml]: (same for the other case, btw)

Comment (FernandoRomera) [reply mgmtsystem_audit/data/audit_automated_actions.xml]: @ivantodorovich 
Thanks.

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**